server/controllers/stock/reports/stock/expiration_report.js

```diff
@@ -47,10 +47,10 @@ async function stockExpirationReport(req, res, next) {
     const lots = await stockCore.getLotsDepot(options.depot_uuid, options);
 
     // get the lots that are "at risk"
-    const risky = lots.filter(lot => (lot.S_RISK < 0 && lot.lifetime > 0));
+    const risky = lots.filter(lot => (lot.at_risk && lot.lifetime > 0));
```

Comment:
So, I think this report is actually looking for "at risk of expiration".  This "at_risk" flag is "at_risk_of_stock_out".  The correct flag you are looking for is `lot.near_expiration` which will filter on lots that will expire before teh CMM says they will be consumed.  We could make that clearer.

Reply:
So, I think you can just change this to:
```js
 const risky = lots.filter(lot => lot.flags.near_expiration);
 ```

Reply:
Done.

However, I'm still seeing duplication of the flags in the lot object as well as in log.flags.